guides/contribution.md

```diff
@@ -0,0 +1,119 @@
+# Contribution guide
+
+Welcome to the Mística Design System contribution guide for external teams.
+
+As an external team, you have the opportunity to contribute to the Mística Design System by sharing your ideas and expertise.
+Whether you are a designer, developer, or product manager, your contributions can help us improve the system and make it more useful for a wider range of users.
+
+This guide is designed to help you understand how you can contribute to Mística, what kind of contributions we are looking for, and how to make sure your contributions are consistent with our way of working.
+
+We encourage you to read this guide carefully and get in touch with us if you have any questions or feedback.
+
+## Index
+
+- [Proposal](#proposals)
+- [Creation of an issue based on the proposal](#issues)
+- [Updating or creating the specs needed](#specifications)
+- [Development and final review](#from-design-to-development)
+
+## Contribution journey
+
+![Contribution journey](../.github/resources/guides/contribution/contribution_journey.svg)
+
+## Proposals
+
+Before submitting a proposal, we encourage you to review:
+
+* Our existing components and guidelines to ensure that your proposal aligns with our design principles and 
+* There's no other discussion already covering the same topic. You can find all of our design guidelines and documentation on [Telefónica's brand factory](https://brandfactory.telefonica.com/d/iSp7b1DkYygv/n-a#/get-started/what-is-mistica).
+* It's not covered by the existing [Component specs](https://www.figma.com/files/1125734703130062955/project/27955986/Component-Specs?fuid=1111936175780412673) 
+* The technical viability to include the proposal in your product (A validated PRD, or confirmation that backend can serve this information)
+
```

Comment:
:robot: OpenAI

The paragraph provides clear criteria for submitting a proposal. However, it would be helpful to provide more detail on what is expected in each criterion.
```suggestion
Before submitting a proposal, we encourage you to review our existing components and guidelines to ensure that your proposal aligns with our design principles. Additionally, please ensure that there is no other discussion already covering the same topic. You can find all of our design guidelines and documentation on [Telefónica's brand factory](https://brandfactory.telefonica.com/d/iSp7b1DkYygv/n-a#/get-started/what-is-mistica). Please also ensure that the proposed change is not covered by the existing [Component specs](https://www.figma.com/files/1125734703130062955/project/27955986/Component-Specs?fuid=1111936175780412673) and that it is technically viable to include the proposal in your product (A validated PRD, or confirmation that backend can serve this information).
```

<!-- This is an auto-generated comment by OpenAI -->